src/components/ModalEdit/index.js

```diff
@@ -0,0 +1,133 @@
+import { useContext } from "react";
+import DatePicker from "react-datepicker";
+import { Modal, Button } from "react-bootstrap";
+import DoctorsContext from "../../contexts/DoctorsContext";
+
+const ModalEdit = ({
```

Comment:
and why does this component have no states and no functions of its own? everything comes through a prop